lib/assessment/code_feature_extractor.py

```diff
@@ -198,41 +198,19 @@ def flatten_conditional_paths(self, conditional):
   # Returns a dict containing the object and the property being operated on.
   def member_expression_helper(self, expression):
     # Get object and property names
-    if expression.object.type == "Literal":
-      member_expression = {"object": expression.object.raw, "property": expression.property.name}
-    elif expression.object.type == "Identifier":
+    if expression.object.type == "Identifier":
       member_expression = {"object": expression.object.name, "property": expression.property.name}
     else:
-      print("member expression outlier", expression) # DEBUG
+      logging.info("member expression outlier", str(expression)) # DEBUG
     return member_expression
 
-  # Helper function to determine the type of an object (sprite, shape, or text)
-  # Returns the object type as a string
-  def object_type_helper(self, node):
-    expressions = []
-    if node.type == 'ExpressionStatement' and node.expression.type == 'CallExpression':
-      expressions.append(node.expression)
-    elif node.type == 'VariableDeclaration':
-      for declaration in node.declarations:
-        if declaration.init.type == 'CallExpression':
-          expressions.append(declaration.init)
-    obj_types = []
-    for expression in expressions:
-      if expression.callee.name in shape_functions:
-        obj_types.append("shapes")
-      elif expression.callee.name == 'text':
-        obj_types.append("text")
-      elif expression.callee.name == 'createSprite':
-        obj_types.append("sprites")
-    return obj_types
-
   # Helper function to parse unary functions. Returns a float combining the
   # operator and number
   def unary_expression_helper(self, expression):
     if expression.argument.type == "Literal":
       return float(expression.operator + expression.argument.raw)
     else:
-      print("unary expression outlier", expression) #DEBUG
+      logging.info("unary expression outlier", str(expression)) #DEBUG
```

Comment:
They don't print by default. You have to change the logger settings to print info level logs to either a file or stdout. I left those in there so that if we're trying to diagnose a statement that isn't running through the pipeline correctly, we can switch the log level to info and see what ends up in those outlier paths.